docker-compose.yml

```diff
@@ -243,24 +243,22 @@ services:
     profiles:
       - monitoring
     restart: unless-stopped
-    image: docker.io/jaegertracing/all-in-one:1.48
-    environment:
-      COLLECTOR_ZIPKIN_HOST_PORT: 9411
-      COLLECTOR_OTLP_ENABLED: "true"
-      SPAN_STORAGE_TYPE: badger
-      BADGER_EPHEMERAL: "false"
-      BADGER_DIRECTORY_VALUE: /badger/data
-      BADGER_DIRECTORY_KEY: /badger/key
+    image: docker.io/jaegertracing/jaeger:2.4.0
```

Comment:
Does this leave out the UI which is included in the all-in-one image?

Reply:
No, ui is still included. It seems for version 2 the all-in-one is included in this image